test/functional/FunctionalCodeActionsSpec.hs

```diff
@@ -221,11 +221,12 @@ spec = describe "code actions" $ do
       -- ignore the first empty hlint diagnostic publish
       [_,diag:_] <- count 2 waitForDiagnostics
 
-      if ghcVersion == GHC86
-        then
-          liftIO $ diag ^. L.message `shouldSatisfy` T.isPrefixOf "Could not load module \8216Data.Text\8217"
-         else
-          liftIO $ diag ^. L.message `shouldSatisfy` T.isPrefixOf "Could not find module ‘Data.Text’"
+      let prefixes = [ "Could not load module `Data.Text'" -- Windows && GHC >= 8.6
+                     , "Could not find module `Data.Text'" -- Windows
+                     , "Could not load module ‘Data.Text’" -- GHC >= 8.6
+                     , "Could not find module ‘Data.Text’"
+                     ]
```

Comment:
Shouldn't we have some uniform way of dealing with the set of quotes? Perhaps run the expected string through a normaliser and then match on the result?